apps/e2e/ag-grid-storybook/src/app/data-entry-grid/data-entry-grid.component.html

```diff
@@ -0,0 +1,14 @@
+<h1>Data Entry Grid</h1>
+
+<ng-container *ngIf="active$ | async as active">
+  <div *ngFor="let dataSet of dataSets" class="sky-margin-stacked-md">
+    <sky-ag-grid-wrapper [id]="dataSet.id">
+      <ag-grid-angular
+        class="sky-ag-grid-editable"
+        [gridOptions]="gridOptions[dataSet.id]"
+      ></ag-grid-angular>
+    </sky-ag-grid-wrapper>
+  </div>
```

Comment:
Multiple grids stacked with a couple different scenarios allow for more coverage in fewer screenshots.